adapter/CMakeLists.txt

```diff
@@ -21,6 +21,13 @@ endif()
 
 if(HERMES_ENABLE_VFD)
   if(HERMES_ENABLE_WRAPPER)
+    # The Hermes VFD cannot be installed into the same directory as libhermes.so
+    # because of the way dynamic plugin loading works in HDF5. When searching
+    # for a VFD, HDF5 will dlopen and dlclose every shared library in
+    # HDF5_PLUGIN_PATH looking for a valid VFD, and when it goes through this
+    # process with libhermes.so it resets some statically initialized variables.
```

Comment:
I wish you could split the setence for clarity.
```
When searching for a VFD, HDF5 will dlopen and dlclose every shared library in
HDF5_PLUGIN_PATH looking for a valid VFD. When HDF5 goes through this
process with libhermes.so, it resets some statically initialized variables.
```

Reply:
Done.